package.json

```diff
@@ -23,8 +23,9 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@project-serum/anchor-cli": "^0.17.0",
     "@project-serum/anchor": "^0.17.0",
+    "@project-serum/anchor-cli": "^0.17.0",
```

Comment:
Since I bumped this in the cargo this should also get bumped to 0.18, but I can do this separately